pkg/storage/edv/formatprovider/formatprovider.go

```diff
@@ -101,22 +145,34 @@ func (p FormatProvider) Close() error {
 }
 
 type formatStore struct {
-	underlyingStore   storage.Store
-	documentProcessor DocumentProcessor
-
-	marshal marshalFunc
+	underlyingStore          storage.Store
+	documentProcessor        DocumentProcessor
+	macCrypto                *MACCrypto
+	indexKeyMACBase64Encoded string
+	marshal                  marshalFunc
+	randomBytesFunc          generateRandomBytesFunc
 }
 
 func (s formatStore) Put(k string, v []byte) error {
```

Comment:
Why isn't formatStore a pointer receiver (i.e., *formatStore)?

when in doubt, use a pointer receiver.

Reply:
Yes, that's a mistake. I had originally fixed it in this PR (you'll see it if you go back a few force-commits), but since I'll be moving this entire file per https://github.com/hyperledger/aries-framework-go/pull/2263#discussion_r512089148, I figured that it would make more sense to keep this PR small and fix it in the next one. I'll make a note in that issue to fix it so that I don't miss it.